config/cli.go

```diff
@@ -154,32 +156,40 @@ type CLIConfig struct {
 	L2Host string
 }
 
-func ReadCLIConfig(ctx *cli.Context) (*CLIConfig, error) {
-	cfg := &CLIConfig{
-		AdminPort: ctx.Uint64(AdminPortFlagName),
+type ForkTOMLConfig struct {
+	L1ForkHeight uint64   `toml:"l1_fork_height"`
+	Network      string   `toml:"network"`
+	Chains       []string `toml:"chains"`
 
-		L1Port:         ctx.Uint64(L1PortFlagName),
-		L2StartingPort: ctx.Uint64(L2StartingPortFlagName),
+	InteropEnabled bool `toml:"interop_enabled"`
+}
 
-		InteropAutoRelay: ctx.Bool(InteropAutoRelayFlagName),
-		InteropDelay:     ctx.Uint64(InteropDelayFlagName),
+type TOMLConfig struct {
+	AdminPort uint64 `toml:"admin_port"`
 
-		LogsDirectory: ctx.String(LogsDirectoryFlagName),
+	L1Port         uint64 `toml:"l1_port"`
+	L2StartingPort uint64 `toml:"l2_starting_port"`
 
-		L1Host: ctx.String(L1HostFlagName),
-		L2Host: ctx.String(L2HostFlagName),
-	}
+	InteropAutoRelay bool   `toml:"interop_autorelay"`
+	InteropDelay     uint64 `toml:"interop_delay"`
 
```

Comment:
TOMLConfig and ForkTOMLConfig should be combined with the ForkCLIConfig and CLIConfig. I think we can just add struct tags to the CLIConfig and ForkCLIConfig.